pkg/pocket/creator/creator.go

```diff
@@ -77,6 +77,19 @@ func (p PocketClient) Start(ctx context.Context, _ interface{}, clientNodes []cl
 	if len(clientNodes) > 1 {
 		cfg.DSN2 = makeDSN(clientNodes[1].Address())
 	}
+	// In the DM case, there are 3 client nodes needed (2 for upstream MySQL and 1 for downstream TiDB).
+	if len(clientNodes) > 2 {
+		cfg.DSN3 = makeDSN(clientNodes[2].Address())
+	}
+
+	switch cfg.Mode {
+	case "dm":
+		err := dmCreateSourceTask(clientNodes)
+		if err != nil {
```

Comment:
We can simplify this to an if statement `if cfg.Mode == "dm"`

Reply:
simplified in https://github.com/pingcap/tipocket/pull/220/commits/01a18e6682c11bb4ab2b1cec1eea349304badf81.